Filters/exclusions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/670
+||alitems.com^
```

Comment:
I guess we already have it https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/fix/670_alitems.com/Filters/exclusions.txt#L135

Reply:
Hm. Weird. Checked nslookup before making PR.

Reply:
It seems corelibs bug.